docs/pages/installation/minikube.md

```diff
@@ -59,16 +59,26 @@ Perform the following steps in order:
 
 ### 2. Install Meshery
 
-Follow the [installation steps](/guides/mesheryctl) to setup the mesheryctl CLI and install Meshery.
+Follow the [installation steps](/installation/quick-start) to setup the mesheryctl CLI and install Meshery.
 
-**Note**: If you are using docker driver, after completing meshery installation steps execute below command to establish connectivity between Meshery and Kubernetes server :
+**Users using docker driver**: 
+
+After completing meshery installation steps execute below command to establish connectivity between Meshery and Kubernetes server :
```

Comment:
```suggestion
After completing meshery installation steps execute below command to establish connectivity between Meshery and Kubernetes server:
```

Reply:
Nice. We'll want to capitalize "Meshery" and refer to the "Kubernetes cluster"